build.sh

```diff
@@ -225,17 +225,16 @@ if [ "$clangTidy" -eq 1 ] && [ `command -v clang-tidy` ]; then
         #clang-tidy -p $compile_json $cpp -checks='*' -header-filter='.*'
 
         # Don't check header, don't check google/llvm coding conventions
-        echo "$count/$total"
```

Comment:
I agree. There's an easy solution that doesn't require a lot (or any) synchronization: we can use GNU parallel (xargs would parallelize but also mangle the output). I'll submit a PR.